src/shared/UniversalRender.jsx

```diff
@@ -351,9 +350,9 @@ async function universalRender({
                 onchain.content[key]['stats'] = contentStats(
                     onchain.content[key]
                 );
-                onchain.content[key]['active_votes'] = onchain.content[key][
-                    'active_votes'
-                ].filter(vote => vote.voter === offchain.account);
+                //onchain.content[key]['active_votes'] = onchain.content[key][
+                //    'active_votes'
+                //].filter(vote => vote.voter === offchain.account);
```

Comment:
This feature is used to reduce page size drastically by removing (potentially 1000's of) vote records which are not relevant to the logged in user; would be ideal to preserve it if at all possible.

Reply:
cool, i'll work this part back in.

Reply:
actually what if we excluded these vote records for anon users, and then for logged-in users we request that data on the client side?

Reply:
When a client loads a SSR page do they immediately request `get_state` again? If yes: then we could probably trim all vote records here, because `get_state` will return the whole set anyway (w/o trimming). It would probably result in visual flashing however. If no: then they would probably need to; I'm not sure there's any other call to get this data (currently).

Reply:
the client ends up doing another `get_state` anyways

Reply:
ok i've excluded all vote data from the SSR response & i don't notice any visual flashing on page loads.